### PR TITLE
Allow secret retrieval from environment variables

### PR DESCRIPTION
**Pull request by guillett**
Depends on #46.

### CODE REVIEW

File: modules/mesaides/files/ma-web.conf [hunk 1]
```diff
@@ -11,4 +11,11 @@ setuid main
 
 chdir /home/main/mes-aides-ui
 
-exec node server
+script
+    set -a
+    if [ -f /home/main/.secrets ]
+    then
+         . /home/main/.secrets
```
Comment (MattiSG) [inline modules/mesaides/files/ma-web.conf]: Prefer `source` over `.`, they are [synonymous](https://askubuntu.com/questions/25488/what-is-the-difference-between-source-and-in-bash) but `source` is muuuuuch more readable.

File: modules/mesaides/files/ma-web.conf [hunk 2]
```diff
@@ -11,4 +11,11 @@ setuid main
 
 chdir /home/main/mes-aides-ui
 
-exec node server
+script
```
Comment (MattiSG) [inline modules/mesaides/files/ma-web.conf]: > Why is this `script` wrapper required while it wasn't before?

File: modules/mesaides/files/ma-web.conf [hunk 3]
```diff
@@ -11,4 +11,11 @@ setuid main
...
+    set -a
+    if [ -f /home/main/.secrets ]
```
Comment (MattiSG) [inline modules/mesaides/files/ma-web.conf]: Use a more specific name. I agree we can assume to be alone on the machine, but still, a `.mes-aides-secrets` would make forensics much easier :wink:

File: modules/mesaides/files/ma-web.conf (hunk 1 above)
Comment (guillett) [inline modules/mesaides/files/ma-web.conf]: Yes they are synonymous but only in `bash`not in shell. `source` is not working is an `upstart` job file.

File: modules/mesaides/files/ma-web.conf (hunk 2 above)
Comment (guillett) [inline modules/mesaides/files/ma-web.conf]: A single `exec` command can be specified directly but multiple commands have to be wrapped in a `script` section, cf. http://upstart.ubuntu.com/wiki/Stanzas#exec.

File: modules/mesaides/files/ma-web.conf (hunk 1 above)
Comment (guillett) [inline modules/mesaides/files/ma-web.conf]: Test done on `vps` instance:
I replaced `.` with `source` and restarted nginx (cf. errors) and replaced `source` with `.` (cf. UpTimeRobot requests after the errors).

```
root@vps274040:~# tail -n 20 /var/log/upstart/ma-web.log
::ffff:127.0.0.1 - - [15/Jun/2017:06:50:01 +0000] "GET / HTTP/1.1" 200 7630 "-" "curl/7.35.0"
::ffff:127.0.0.1 - - [15/Jun/2017:06:50:12 +0000] "HEAD / HTTP/1.0" 200 7630 "https://vps.mes-aides.gouv.fr" "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
/proc/self/fd/9: 5: /proc/self/fd/9: source: not found
Mes Aides server listening on port 8000, in production mode, expecting to be deployed on https://mes-aides.gouv.fr
::ffff:127.0.0.1 - - [15/Jun/2017:06:53:14 +0000] "GET / HTTP/1.1" 200 7630 "-" "curl/7.35.0"
::ffff:127.0.0.1 - - [15/Jun/2017:06:53:23 +0000] "HEAD / HTTP/1.0" 200 7630 "https://vps.mes-aides.gouv.fr" "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"
::ffff:127.0.0.1 - - [15/Jun/2017:06:54:13 +0000] "GET / HTTP/1.1" 200 7630 "-" "curl/7.35.0"
::ffff:127.0.0.1 - - [15/Jun/2017:06:54:22 +0000] "HEAD / HTTP/1.0" 200 7630 "https://vps.mes-aides.gouv.fr" "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"
::ffff:127.0.0.1 - - [15/Jun/2017:06:55:13 +0000] "GET / HTTP/1.1" 200 7630 "-" "curl/7.35.0"
::ffff:127.0.0.1 - - [15/Jun/2017:06:55:21 +0000] "HEAD / HTTP/1.0" 200 7630 "https://vps.mes-aides.gouv.fr" "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"
```

**Pull request merged**